package.json

```
@@ -1,8 +1,9 @@
 {
   "name": "click-should-be-intercepted-for-navigation",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

```suggestion
  "version": "1.0.0",
```

the version bump happens automatically as part of my publish script